core/src/main/java/org/nervos/ckb/methods/type/Script.java

```diff
@@ -35,17 +35,8 @@ public Script(String codeHash, List<String> args, String hashType) {
   }
 
   public String scriptHash() {
```

Comment:
Will uniform naming be more appropriate? like `serialization`

Reply:
I think serialization just includes itself and script hash which can be lock hash and type hash includes serialization and blake2b.

So I think serilization can not explain all things of this method.